connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -234,85 +238,98 @@ export async function syncOneFile(
         await fs.unlink(pdf_path);
       }
     }
+  } else if (isGoogleDriveSpreadSheetFile(file)) {
+    // If Google Spreadsheet FF is not enabled, it returns false.
+    const isSupported = await syncSpreadSheet(oauth2client, connectorId, file);
+    if (!isSupported) {
+      return false;
+    }
   } else {
-    // We do not support this file type
+    // We do not support this file type.
     return false;
   }
 
-  documentContent = documentContent?.trim();
-
-  const content = await renderDocumentTitleAndContent({
```

Comment:
No changes here, just moved in a if block.

💡 Better reviewed with hidden whitespaces.